ONSdigital/census-rm-toolbox
# ADDRESS_DELTA - Create file of case_ids from invalid address UPRNs

# Motivation and Context
Need to be able to remove addresses that have been discovered so that we do not waste resources contacting invalid addresses

# What has changed
New feature to call Case API to find case_ids of UPRNs in a file
Case_ids are then moved into new CSV file and marked as ADDRESS_DELTA to show they are invalid
File is uploaded to GCP bucket 
New alias

# How to test?
Create dummy file of UPRNs locally
Set up GCP env
Run ATs in GCP to create cases
Run bulk_processing feature in toolbox against feature
Check file of case_ids and reason has been created in GCP bucket
Search for cases selected against UPRNs in toolbox and check address_invalid is marked as t

# Links
https://trello.com/c/lsVwLMnd/1021-address-frame-delta-mark-cases-address-not-valid-3
https://collaborate2.ons.gov.uk/confluence/pages/viewpage.action?spaceKey=SDC&title=Delta+Sample+Frames

Run Book: https://collaborate2.ons.gov.uk/confluence/display/SDC/Find+Invalid+Address+Case+ID%27s+by+UPRN+-+ADDRESS_DELTA
